modules/products/bamboo/variables.tf

```diff
@@ -115,11 +145,15 @@ variable "local_agent_chart_path" {
   default = ""
 }
 
+# TODO: This variable was added to accept pcv claim, however, based on the current
```

Comment:
nit: Just for consistency reason, all other products have removed this variable. 
And typo `pcv`.